docs/docs/lib/node.mdx

```diff
@@ -45,59 +111,45 @@ app.use(function(req, res, next) {
 });
 ```
 
-Then, you can access the GrowthBook instance from any route:
-
-```js
-app.get("/", (req, res) => {
-  const gb = req.growthbook;
-
-  // Boolean on/off flag
-  if (gb.isOn("my-feature")) {
-    // Do something
-  }
-
-  // String/Number/JSON flag
-  const value = gb.getFeatureValue("my-string-feature", "fallback");
-  console.log(value);
-})
-```
+The `GrowthBook` class is identical to the one used in client-side environments. Check out the [/lib/js](Client-side JavaScript SDK docs) for more info on all of the settings and methods available.
```

Comment:
Syntax is reversed for the link:
`[/lib/js](Client-side)` -> `[Client-side JS SDK docs](/lib/js)`